fedoragsearch-transforms/foxmlToSolr.xslt

```diff
@@ -91,28 +91,28 @@
   <xsl:include href="/var/lib/tomcat7/webapps/fedoragsearch/WEB-INF/classes/config/index/FgsIndex/islandora_transforms/hierarchy.xslt"/>
   -->
 
-  <xsl:include href="/vhosts/fedora/tomcat/webapps/fedoragsearch/WEB-INF/classes/fgsconfigFinal/index/FgsIndex/islandora_transforms/DC_to_solr.xslt"/>
-  <xsl:include href="/vhosts/fedora/tomcat/webapps/fedoragsearch/WEB-INF/classes/fgsconfigFinal/index/FgsIndex/islandora_transforms/RELS-EXT_to_solr.xslt"/>
-  <xsl:include href="/vhosts/fedora/tomcat/webapps/fedoragsearch/WEB-INF/classes/fgsconfigFinal/index/FgsIndex/islandora_transforms/RELS-INT_to_solr.xslt"/>
-  <xsl:include href="/vhosts/fedora/tomcat/webapps/fedoragsearch/WEB-INF/classes/fgsconfigFinal/index/FgsIndex/islandora_transforms/FOXML_properties_to_solr.xslt"/>
-  <xsl:include href="/vhosts/fedora/tomcat/webapps/fedoragsearch/WEB-INF/classes/fgsconfigFinal/index/FgsIndex/islandora_transforms/datastream_info_to_solr.xslt"/>
-  <!--<xsl:include href="/vhosts/fedora/tomcat/webapps/fedoragsearch/WEB-INF/classes/fgsconfigFinal/index/FgsIndex/islandora_transforms/MODS_to_solr.xslt"/>-->
-  <xsl:include href="/vhosts/fedora/tomcat/webapps/fedoragsearch/WEB-INF/classes/fgsconfigFinal/index/FgsIndex/islandora_transforms/slurp_all_MODS_to_solr.xslt"/>
-  <xsl:include href="/vhosts/fedora/tomcat/webapps/fedoragsearch/WEB-INF/classes/fgsconfigFinal/index/FgsIndex/islandora_transforms/slurp_all_ead_to_solr.xslt"/>
-  <xsl:include href="/vhosts/fedora/tomcat/webapps/fedoragsearch/WEB-INF/classes/fgsconfigFinal/index/FgsIndex/islandora_transforms/EACCPF_to_solr.xslt"/>
-  <xsl:include href="/vhosts/fedora/tomcat/webapps/fedoragsearch/WEB-INF/classes/fgsconfigFinal/index/FgsIndex/islandora_transforms/TEI_to_solr.xslt"/>
-  <xsl:include href="/vhosts/fedora/tomcat/webapps/fedoragsearch/WEB-INF/classes/fgsconfigFinal/index/FgsIndex/islandora_transforms/text_to_solr.xslt"/>
-  <!--<xsl:include href="/vhosts/fedora/tomcat/webapps/fedoragsearch/WEB-INF/classes/fgsconfigFinal/index/FgsIndex/islandora_transforms/XML_to_one_solr_field.xslt"/>-->
-  <xsl:include href="/vhosts/fedora/tomcat/webapps/fedoragsearch/WEB-INF/classes/fgsconfigFinal/index/FgsIndex/islandora_transforms/XML_text_nodes_to_solr.xslt"/>
-  <xsl:include href="/vhosts/fedora/tomcat/webapps/fedoragsearch/WEB-INF/classes/fgsconfigFinal/index/FgsIndex/islandora_transforms/MADS_to_solr.xslt"/>
-  <xsl:include href="/vhosts/fedora/tomcat/webapps/fedoragsearch/WEB-INF/classes/fgsconfigFinal/index/FgsIndex/islandora_transforms/WORKFLOW_to_solr.xslt"/>
-  <xsl:include href="/vhosts/fedora/tomcat/webapps/fedoragsearch/WEB-INF/classes/fgsconfigFinal/index/FgsIndex/islandora_transforms/slurp_all_chemicalML_to_solr.xslt"/>
+  <xsl:include href="/var/lib/tomcat7/webapps/fedoragsearch/WEB-INF/classes/fgsconfigFinal/index/FgsIndex/islandora_transforms/DC_to_solr.xslt"/>
```

Comment:
I don't think we want to commit any of these changes to this file to the code base.  This is very much tied to the Ubuntu base box on TRACE vagrant.  All changes from this file should not be included.

Not sure how you want to do this, but you could I guess just do a soft reset back to the hash prior to this commit, then use reset to untrack file, then recommit.

Something like this:

https://stackoverflow.com/questions/12481639/remove-files-from-git-commit

@CanOfBees what do you think?

Reply:
A word about the template.

http://www.loc.gov/standards/mods/v3/mods-3-5.xsd

****************************************************
*   Top Level Element <name>                             *
****************************************************        
--><xs:element name="name" type="nameDefinition"/>
<!-- -->
<xs:complexType name="nameDefinition">
<xs:choice>
<!--  this choice is introduced in version 3.5, with the introduction of <etal>.  
      There are two "choices", one without <etal> (the old way) and one with <etal>.
      Choice one. The old way.
-->
<xs:choice minOccurs="0" maxOccurs="unbounded">
  <xs:element ref="namePart"/>
  <xs:element ref="displayForm"/>
  <xs:element ref="affiliation"/>
  <xs:element ref="role"/>
  <xs:element ref="description"/>
</xs:choice>

Because the MODS online schema shows namePart and displayForm at the same level,
I am treating them as being at the same level in the template.

Reply:
Test notes and screen shots in JIRA TRAC-886 ticket.
I believe this is ready for review.
TRACE(TRAC-886) loads utk-fedora-fedoragsearch-solr-config(TRAC-886a)
for testing with vm.
@markpbaggett @CanOfBees

Reply:
@cdeaneGit , apologies.  I think I didn't get what you were asking.  Here your shifting the paths to /var/lib/tomcat7, but that's only right if we're talking about Vagrant.  Historically, we haven't changed this sort of thing.